Documentation.app/Contents/MacOS/First steps.playground/contents.xcplayground

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <playground version='6.0' target-platform='ios' display-mode='raw'>
     <pages>
+        <page name='Xcode Playground → nef Playground'/>
+        <page name='Playground Book with 3rd-party libraries'/>
         <page name='How to export my snippets?'/>
-        <page name='How to use Carbon API?'/>
+        <page name='How to use carbon api?'/>
```

Comment:
Carbon should always be capitalized. API should always be uppercased, as it is an acronym.

Reply:
Your suggestions are the opposite, rigth?

Reply:
No, you have changed it and I am asking to revert it to how it was before.

Reply:
Ahhhhhh!!! I saw it as a “commit suggestion” - it should be happening during merge! Sure, thanks :)

Reply:
Ok, I see what happens, it is the "place" where playground links its pages, and it continues in lowercased, but as it is not case-sensitive it is fine. Anyway, I have updated it.